go.mod

```diff
@@ -199,3 +199,7 @@ require (
 )
 
 replace github.com/robfig/cron/v3 => github.com/unionai/cron/v3 v3.0.2-0.20210825070134-bfc34418fe84
+
+replace github.com/flyteorg/flyteidl => github.com/MorpheusXAUT/flyteidl v0.24.22-0.20220426074538-c412ba26cf64
+
+replace github.com/flyteorg/flytepropeller => github.com/MorpheusXAUT/flytepropeller v0.16.48-0.20220426142632-4fdc51f2f6bb
```

Comment:
I suppose this was only for testsing . Can we remove this

Reply:
The PRs to [flyteidl](https://github.com/flyteorg/flyteidl/pull/287)/[flytepropeller](https://github.com/flyteorg/flytepropeller/pull/429) are still not merged yet, so I'm still relying on this for changes made to protos/core spec. Will remove and replace with updated versions once the respective PRs have been merged :+1: